components/form/FAutocompleteInput.vue

```diff
@@ -354,7 +354,7 @@ function handleInput(e: InputEvent) {
 }
 
 function isValidMatch() {
-  return !!currentOptionMatched.value;
+  return !inputValue.value || !!currentOptionMatched.value;
```

Comment:
All the issue mentioned on the daily were just related to that? 🥲

Reply:
No, not all, some are still here